src/k2/module/k23si/Module.cpp

```diff
@@ -191,23 +191,65 @@ seastar::future<> K23SIPartitionModule::_registerVerbs() {
 
     RPC().registerRPCObserver<dto::K23SIReadRequest, dto::K23SIReadResponse>
     (dto::Verbs::K23SI_READ, [this](dto::K23SIReadRequest&& request) {
-        return handleRead(std::move(request), FastDeadline(_config.readTimeout()));
+        OperationLatencyReporter reporter(_readOps, _readLatency);
+        #ifdef GET_LATENCY
+            reporter.startOperation();
```

Comment:
I don't think there should be a separate startOperation method, the work should be done in the constructor. Also maybe the compilation flag can be inside the Reporter class, and we can always count the number of ops even if the flag is off.

Reply:
Discussed offline with Justin. Decided to add extra compilation flag FORCE_GET_LATENCY when GET_LATENCY flag is not defined.

Reply:
done